packages/communication-react/scripts/verifyTypeTrimmedDeclaration.js

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Verify the 'types' field is correct in the package.json based on the supplied release type parameter.
+ *
+ * @command verify - this is the default. This checks the field is correct.
+ * @command update - this can be used to update the field to the correct string.
+ *
+ * @param releaseCandidate - this must be 'public', 'beta' or 'alpha'.
+ *
+ * @examples
+ *   > node .\verifyTypeTrimmedDeclaration.js verify --rc public
+ *   > node .\verifyTypeTrimmedDeclaration.js verify --rc beta
+ *   > node .\verifyTypeTrimmedDeclaration.js update --rc public
+ *
+ * @remarks
+ * Passing the following parameters should result in the following checks:
```

Comment:
I'd drop this comment. It repeats what the code says, without adding why. It's like to bitrot.

Reply:
updated